libs/gi/db/src/Database/migrate.ts

```diff
@@ -208,6 +208,7 @@ export function migrateGOOD(good: IGOOD & IGO): IGOOD & IGO {
         description: `Generated team due to database migration for GO version 10`,
         enemyOverride,
         teamCharIds,
+        ...{ lastEdit: 0 },
```

Comment:
```suggestion
        lastEdit: 0,
```
Why does this need to be encapsulated?

Reply:
MB. It doesn't need to.